sklearn/inspection/_permutation_importance.py

```diff
@@ -78,9 +78,11 @@ def permutation_importance(estimator, X, y, scoring=None, n_repeats=5,
         `-1` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
-    random_state : int, RandomState instance, or None, default=None
+    random_state : int, RandomState instance or None (default)
```

Comment:
```suggestion
    random_state : int, RandomState instance, default=None
```

Reply:
I know it's not important, but if you have the time could you explain why you suggested this change?

Reply:
The issue was opened before that we change the documentation guideline which is accessible here:

https://scikit-learn.org/dev/developers/contributing.html#guidelines-for-writing-documentation

I will edit the issue then.

Reply:
I updated the issue @Malesche Thanks for pointing this out.

Reply:
Thanks!